app/pages/Accounts/AccountDetailsPage/StakingDetails/StakingDetails.tsx

```diff
@@ -0,0 +1,226 @@
+import {
+    AccountBakerDetails,
+    AccountBakerDetailsV1,
+    AccountDelegationDetails,
+} from '@concordium/node-sdk';
+import { isReduceStakePendingChange } from '@concordium/node-sdk/lib/src/accountHelpers';
+import React, { PropsWithChildren } from 'react';
+import RegisteredIcon from '@resources/svg/logo-checkmark.svg';
+import Label from '~/components/Label';
+import Card from '~/cross-app-components/Card';
+import Loading from '~/cross-app-components/Loading';
+import { displayAsCcd } from '~/utils/ccd';
+import { useConsensusStatus } from '~/utils/dataHooks';
+import { toFixed } from '~/utils/numberStringHelpers';
+import { hasDelegationProtocol } from '~/utils/protocolVersion';
+import {
+    dateFromStakePendingChange,
+    getFormattedDateString,
+} from '~/utils/timeHelpers';
+import { displayPoolOpen } from '~/utils/transactionFlows/configureBaker';
+import { displayDelegationTarget } from '~/utils/transactionFlows/configureDelegation';
+
+import styles from './StakingDetails.module.scss';
+
+interface ValueProps {
+    title: string;
+    value: string;
+}
+
+function Value({ title, value }: ValueProps) {
+    return (
+        <div className="mB20:notLast">
```

Comment:
love it!

Reply:
I don't know if you're sarcastic, but it makes sense here as we don't know whether it's the last in line or not 😉